Content.Client/UserInterface/Systems/Chat/Widgets/ChatBox.xaml.cs

```diff
@@ -7,7 +9,9 @@
 using Robust.Client.UserInterface;
 using Robust.Client.UserInterface.Controls;
 using Robust.Client.UserInterface.XAML;
+using Robust.Shared;
 using Robust.Shared.Audio;
```

Comment:
Using directive is unnecessary.
```suggestion
using Robust.Client.UserInterface.XAML;
using Robust.Shared.Audio;
```